File: pruebas/prueba.py

def getName():
    respuesta = 'siguiente'
    return respuesta

print("Hello world")
print(getName())

palabra = 'zixmtkozy'
decodificada = ''
for letra in palabra:
    letra_unicode = ord(letra)-ord('a')
    nuevaLetra_unicode = (letra_unicode + 343)%26
    decodificada += chr(ord('a')+nuevaLetra_unicode)
print(decodificada)

File: pruebas/prueba.java

public class prueba {

    public static String getName() {
        String respuesta = "siguiente";
        return respuesta;
    }

    public static void main(String[] args) {
        String[] tokens = {"hola", "chao", "ultimo"};
        String tok = String.join(", ", tokens);
        System.out.println(tok);
        tokens[tokens.length - 1] = null;
        tok = String.join(", ", tokens);
        System.out.println(tok);
    }

}
